R/modspec.R

```diff
@@ -368,7 +368,7 @@ param_re_find <- "\\bparam\\s+\\w+\\s*="
 # please-deprecate
 move_global <- function(x,env) {
   
-  what <- intersect(c("PREAMBLE","MAIN", "ODE", "TABLE", "PRED"),names(x))
+  what <- intersect(c("PREAMBLE","MAIN", "ODE", "TABLE", "EVENT", "PRED"),names(x))
```

Comment:
I updated this, but it's not in use any more (see comment). I'd rather get rid of it at this point, but it may or may not be in scope of this PR.

Reply:
Taking a quick look at `move_global`, I think a dedicated/separate PR would be better.

Reply:
Agree.